homeassistant/components/pyload/strings.json

```diff
@@ -17,6 +17,20 @@
           "port": "pyLoad uses port 8000 by default."
         }
       },
+      "reconfigure_confirm": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "username": "[%key:common::config_flow::data::username%]",
```

Comment:
Is the username needed to reconfigure? Wouldn't this create a new entity at all?
Also, are some properties maybe better of in the options?

Reply:
@erwindouna 👋 

- It is fine to expose all options in re-configure step.
- These are all connection details, so those go into the data/config flow steps. Only behavioral change (non-connection details) go into options. It seems these are all connection details.